src/main/java/frc/robot/Constants.java

```diff
@@ -77,12 +77,22 @@
     // From example code: kF: 1023 represents output value to Talon at 100%, 20660 represents Velocity units at 100% output
     public static final double SHOOTER_KF = 1023.0/20660.0;  // CTRE example = 1023.0/20660.0
 
+    // amount of time to wait for the motors on shooters to speed up, later replaced with checking RPM
+    public static final double SHOOTER_MOTOR_WAIT_TIME = 0.5;
+    // time to wait for the shots from the shooter
+    public static final double SHOOTER_INTAKE_WAIT_TIME = 0.25;
+    // amount of time after spin-up to do 2 shots
+    public static final double SHOTS_WAIT_TIME = 0.5;
+    // amount of time for the intake to carry the next ball to the chute
+    public static final double INTAKE_WAIT_TIME = 0.5;
```

Comment:
SHOTS_WAIT_TIME and INTAKE_WAIT_TIME are poor names. INTAKE_WAIT_TIME does not even mention shooting.
Maybe use "SHOOTER_BALL1_WAIT_TIME" and "SHOOTER_BALL2_WAIT_TIME"?

Reply:
Also need to update the comments to match the usage.